### PR TITLE
A58 update: Update the status

### PR DESCRIPTION
**Pull request by yousukseung**

### CODE REVIEW

File: A58-client-side-weighted-round-robin-lb-policy.md [hunk 1]
```diff
@@ -2,9 +2,9 @@ A58: `weighted_round_robin` LB policy
 ----
 * Author(s): @yousukseung
 * Approver: @markdroth
-* Status: Draft
-* Implemented in: -
-* Last updated: 2023-02-05
+* Status: Ready for Implementation
+* Implemented in: C-core, Java
+* Last updated: 2023-05-09
```
[the remark this answers is not in the document]
Comment (yousukseung) [reply A58-client-side-weighted-round-robin-lb-policy.md]: Changed to the date when we updated with error_utilization_penalty and the weight formula.

File: A58-client-side-weighted-round-robin-lb-policy.md [hunk 2]
```diff
@@ -2,9 +2,9 @@ A58: `weighted_round_robin` LB policy
 ----
 * Author(s): @yousukseung
 * Approver: @markdroth
-* Status: Draft
-* Implemented in: -
-* Last updated: 2023-02-05
+* Status: Implemented
+* Implemented in: C-core, Java
```
Comment (atollena) [inline A58-client-side-weighted-round-robin-lb-policy.md:6]: Isn't go also implemented? https://github.com/grpc/grpc-go/tree/master/balancer/weightedroundrobin looks like impelmentation of this.

Comment (yousukseung) [reply A58-client-side-weighted-round-robin-lb-policy.md:6]: @dfawley @zasweq we we ready to add Go here?

**Review by markdroth — APPROVED** [review]

File: A58-client-side-weighted-round-robin-lb-policy.md (hunk 2 above)
Comment (zasweq) [reply A58-client-side-weighted-round-robin-lb-policy.md:6]: Yeah it's done. There's a few pending bug fixes and linkage into xDS prs nut other than that it's done.

Comment (yousukseung) [reply A58-client-side-weighted-round-robin-lb-policy.md:6]: Added Go.

File: A58-client-side-weighted-round-robin-lb-policy.md [hunk 3]
```diff
@@ -2,9 +2,9 @@ A58: `weighted_round_robin` LB policy
 ----
 * Author(s): @yousukseung
 * Approver: @markdroth
-* Status: Draft
-* Implemented in: -
-* Last updated: 2023-02-05
+* Status: Ready for Implementation
```
[the remark this answers is not in the document]
Comment (yousukseung) [reply A58-client-side-weighted-round-robin-lb-policy.md]: Updated.

**Pull request merged**